mavros_extras/src/plugins/mount_control.cpp

```diff
@@ -102,6 +102,12 @@ class MountControlPlugin : public plugin::PluginBase {
 		publish_msg.vector.z = ms.pointing_c;
```

Comment:
Ah, it's in centidegrees. Then you need to normalise them. Something like that:
```
auto vec = Eigen::Vector3d(ms.pointing_b, ms.pointing_a, ms.pointing_c) / 100.0;
tf::toMsg(vec, publish_msg.vector);
```

Reply:
Sorry, wrong conversion, should be: `* M_PI / 18000.0`, because vectors should be in radians.

Reply:
I didn't make any conversion since [mavlink message documentation](https://mavlink.io/en/messages/ardupilotmega.html#MOUNT_STATUS) says that it is centidegree.

Reply:
Yes, and ROS convention to use radians almost everywhere, that is why conversion needed.

Reply:
The switch between `pointing_b` and `pointing_a` were on purpose?

Reply:
No, i just copy-pasted them from somewhere. Should be in following order: roll / pitch / yaw.

Reply:
Ok, so b, a, c is correct since a is pitch and b is roll.